contact.html

```diff
@@ -0,0 +1,164 @@
+<html>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<head>
+  <title>Contact</title>
+
+  <link rel="stylesheet" href="css/bootstrap.min.css">
+  <style>
+    #map {
+      height: 350px;
+      /* The height is 400 pixels */
+      width: 100%;
+      /* The width is the width of the web page */
+    }
+  </style>
+  <script>
+    // Initialize and add the map
+    function initMap() {
+      // The location of Uluru
+      const uluru = { lat: 46.7935879, lng: -71.2627805 };
+      // The map, centered at Uluru
+      const map = new google.maps.Map(document.getElementById("map"), {
+        zoom: 15,
+        center: uluru,
+      });
+      // The marker, positioned at Uluru
+      const marker = new google.maps.Marker({
+        position: uluru,
+        map: map,
+      });
+    }
+
+    window.initMap = initMap;
+
+  </script>
+</head>
+
+<body>
+  <nav class="sticky-top navbar navbar-expand-lg bg-light border-bottom border-4">
+    <div class="card-body align-items-end  text-light container-fluid justify-content-center">
+      <ul class="navbar-nav align-items-center">
+        <li class="nav-item active">
+          <a class="navbar-brand" href="/index.htm">Raphaël Doucet</a>
```

Comment:
Ici dans le lien vers votre nom, il manque juste le l pour compléter le lien. /index.html

Reply:
Good catch. And why do you have a home.html page and an index.html too?